src/api.ts

```diff
@@ -201,6 +210,10 @@ export class OpsgenieApi implements Opsgenie {
     return `${this.domain}/settings/users/${userId}/detail`;
   }
 
+  isReadOnly(): boolean {
+    return this.readOnly;
+  }
```

Comment:
To make sure we enforce this setting, could we add a guard on all non-readonly methods in this class? 
`acknowledgeAlert()` and `closeAlert()` should throw an exception if they are somehow called and the readonly flag is true.